pykokkos/lib/constants.py

```diff
@@ -0,0 +1,6 @@
+import math
+
+e = math.e
+pi = math.pi
+inf = math.inf
+nan = math.nan
```

Comment:
this feels a bit lazy, although locally the array API suite didn't seem to complain